### DISCUSSION TITLE
Need help changing owner of vercel

### DISCUSSION THREAD

**Discussion by matan-preql** [vercel/community#4863] [Account Issue] [answered]
### Question

We don't know who is the owner of our Vercel account, and we need to update payment method. need help

**Comment by amyegan** [comment]
Hi @matan-preql. If you go to your Team Settings, then go to the Member tab, you can see a list of team members and their roles. That will let you identify who is an owner of the team.

**Reply by matan-preql** [reply]
Thanks for the response @amyegan , the problem is that we cannot log in the the dashboard, we dont have the account to use.
My guess is that the person who set it up no longer works with us

**Reply by amyegan** [reply]
Do you mean that you can't log in to your Vercel account, or that your account does not have access to the team? If you're able to log in then you should be able to switch to your team from the menu at the top of the page. 

<img width="701" alt="Vercel dashboard navigation" src="https://github.com/vercel/community/assets/8247105/907d9f3f-639a-44c9-9ccb-a1a6f4aa22e3">

**Comment by matan-preql** [comment]
cant log in

On Fri, Nov 17, 2023 at 6:49 PM Amy ***@***.***> wrote:

> Do you mean that you can't log in to your Vercel account, or that your
> account does not have access to the team? If you're able to log in then you
> should be able to switch to your team from the menu at the top of the page.
> [image: Vercel dashboard navigation]
> <https://user-images.githubusercontent.com/8247105/283870354-907d9f3f-639a-44c9-9ccb-a1a6f4aa22e3.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/4863#discussioncomment-7601431>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BCPG67OFBH2J62UUUKZNKJTYE6ITFAVCNFSM6AAAAAA7MSQN3CVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMMBRGQZTC>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*MATAN COHEN*
VP R&D | PREQL <https://www.preql.com/>

**Reply by amyegan** [reply, marked as answer]
Since you can't log in to your personal account, please [submit a case](https://vercel.com/help#issues) to the Customer Success Team. We can't safely discuss account information here in public, but you can find direct channels in the [Vercel Contact Guide](https://vercel.com/guides/how-to-get-vercel-support).
